addons/product_expiry/i18n/product_expiry.pot

```diff
@@ -43,6 +43,11 @@ msgstr ""
 msgid "Alert Time"
 msgstr ""
 
+#. module: product_expiry
+#: model_terms:ir.ui.view,arch_db:product_expiry.report_lot_label_expiry
+msgid "B.b."
```

Comment:
Shouldn't the `"Best before:" and the "Use by:" be deleted ?

Reply:
Nop because I don't want to break the existing translations (needed for those who will still use the old XML version)